src/styles/index.scss

```
@@ -1,3 +1,135 @@
+
+  $main-accent: #060B35 ;
+  $secondary: #616070;
+
 body {
   margin: 0;
 }
+
+.card {
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  width: 198px;
+  height: 406px;
+  flex-shrink: 0;
+  text-align: center;
+  justify-content: center;
+  justify-items: center;
+
+  &__image {
+    margin: 32px 19px 28px;
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+  }
+
+  &__text {
+    width: 166px;
+    text-align: left;
+    margin: 8px 0 0 16px;
+    font-family: Roboto , sans-serif;
+    color: $main-accent;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px; /* 180% */
```

Remove comments everywhere
```suggestion
    line-height: 18px;
```